.github/workflows/ci.yml

```
@@ -1,154 +1,145 @@
-# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-maven
-
-name: Java CI with Maven 
+# CI Workflow for Java Project with Maven and Tycho
+name: Java CI with Maven
 
 on:
-  push:
-    branches: [ master ]
-  pull_request:
-    branches: [ master ]
+	push:
```

actionlint (1.7.4)

could not parse as YAML: yaml: line 5: found character that cannot start any token

(syntax-check)